static/styles/_colors.scss

```diff
@@ -0,0 +1,40 @@
+// You can use it like color: $main_font;
+//main colors
+$red: #EC3944;
+$red_dark: #B6353D;
+$red_light: #F89299;
+//text colors
```

Comment:
• Add black whitelines before each beginning of the section 
• Add whitespace after `//`